docs/drivers/virtualbox.md

```diff
@@ -29,6 +29,8 @@ Options:
  - `--virtualbox-boot2docker-url`: The URL of the boot2docker image. Defaults to the latest available version.
  - `--virtualbox-import-boot2docker-vm`: The name of a Boot2Docker VM to import.
  - `--virtualbox-hostonly-cidr`: The CIDR of the host only adapter.
+ - `--virtualbox-hostonly-nictype`: Host Only Network Adapter Type. Possible values are are '82540EM' (Intel PRO/1000) and 'Am79C973' (PCnet-FAST III)
```

Comment:
To be specific, I think @ehazlett means to add "virtio" to the list of available options here